.ibm/pipelines/env_variables.sh

```diff
@@ -21,20 +24,24 @@ GITHUB_APP_WEBHOOK_URL=aHR0cHM6Ly9zbWVlLmlvL0NrRUNLYVgwNzhyZVhobEpEVzA=
 GITHUB_APP_WEBHOOK_SECRET=$(cat /tmp/secrets/GITHUB_APP_WEBHOOK_SECRET)
 GITHUB_URL=aHR0cHM6Ly9naXRodWIuY29t
 GITHUB_ORG=amFudXMtcWU=
+GH_USER_ID=$(cat /tmp/secrets/GH_USER_ID)
+GH_USER_PASS=$(cat /tmp/secrets/GH_USER_PASS)
+GH_2FA_SECRET=$(cat /tmp/secrets/GH_2FA_SECRET)
 
```

Comment:
```suggestion
GH_RHDH_QE_USER_TOKEN=$(cat /tmp/secrets/GH_RHDH_QE_USER_TOKEN)

```

@gustavolira Please add this to resolve current github test failure.